test/BUILD

```diff
@@ -168,16 +170,6 @@ test_corpus_lsp_tests = pipeline_tests(
     filter = "LSPTests",
 )
 
-test_suite(
-    name = "test_corpus",
-    tests = test_corpus_tests,
-)
-
-test_suite(
-    name = "test_corpus_lsp",
-    tests = test_corpus_lsp_tests,
-)
-
 test_suite(
     name = "test",
```

Comment:
Maybe do all of them for concistwncy?

Reply:
That last test suite is just the fast `bazel test test` target.